YPP-0039.md

```diff
@@ -0,0 +1,26 @@
+# Proposal
+Set the initial offer for all assets in Arbitrum with 18 decimals. Raise the auction limits for all assets.
+
+# Background
+The initial proportion for all assets was set with 6 decimals, when it needs 18, effectively rendering the first two thirds of any auction uneconomical, and compressing the auction into the last third.
+
+# Details
+The [updateAuctions.ts](https://github.com/yieldprotocol/environments-v2/blob/4bf6c0f8a2576da54d660e28679ff3c0104e4603/scripts/governance/update/updateAuctions/updateAuctions.ts) script will be used, with this input:
```

Comment:
Link is broken
https://github.com/yieldprotocol/environments-v2/blob/3bf91f82ffd0205e79d825f7f893745b96afb752/scripts/governance/update/updateAuctions/updateAuctions.ts

Reply:
Fixed